### PR TITLE
feat: better print classlike node comments

### PR DESCRIPTION
**Pull request by evilebottnawi**
Handle comments in `interface` and `trait`

### CODE REVIEW

File: tests/comments/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -2246,6 +2301,53 @@ throw new ClassName /* Comment */(
...
+{
+    // Comment
+}
+
+// Comment
+trait tFoo
+    /* Comment */
+    extends ReflectionMethod
+{
+
```
Comment (evilebottnawi) [inline tests/comments/__snapshots__/jsfmt.spec.js.snap]: Extra empty line is not related to this PR, let's fix it in other PR :+1:

**Comment by evilebottnawi** [comment]
Merge to continue work, feel free to feedback

File: src/comments.js [hunk 2]
```diff
@@ -195,13 +195,33 @@ function handleForComments(
...
-      return true;
+
+    if (followingNode && enclosingNode.extends) {
+      if (!Array.isArray(enclosingNode.extends)) {
+        if (followingNode === enclosingNode.extends) {
+          addDanglingComment(followingNode, comment);
+          return true;
+        }
+      } else {
+        if (
```
Comment (czosel) [inline src/comments.js]: `else if`

Comment (evilebottnawi) [reply src/comments.js]: It will be necessary here to rewrite something here so I will correct in the following PRs
